### PR TITLE
Add github files

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Add github files in order to follow our standards and the good practice to work with opensource.  

## What
Add github files 

## Why
- Following the [opensource guide](https://opensource.guide/) recommendations and checklists for contributors choose a project to contribute with
- Following our standards
- Help the users/contributor track properly an issue and a PR

## How
Applying the same ones defined in: https://github.com/integr8ly/installation/tree/master/.github

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<!-- Describe the current/actual behaviour. -->
+
+## Environment
+
+* Operating system: (E.g RHEL 7.6 )
+* OpenShift version:
+<!-- Run the command `oc version` and add the result here. -->
+* Ansible version:
+<!-- Run the command `ansible --version` and add the result here. -->
+* Project Version/Tag: (E.g release-1.0.1)
```
Comment (camilamacedo86) [inline .github/ISSUE_TEMPLATE.md]: Hi @tremes and @trepel,

Could you help here? Is this env good for this project? Could you review this PR?

Comment (trepel) [reply .github/ISSUE_TEMPLATE.md]: I think it is OK @camilamacedo86

Comment (tremes) [reply .github/ISSUE_TEMPLATE.md]: @camilamacedo86 well I think the environment (and likely other items) description are not necessary here...or simply doesn't make sense. The jobs generated from this repository will always run in a Jenkins and I don't think there will be some releases.

**Comment by tremes** [comment]
Ahh it's already merged:-)

**Pull request merged**